services/sky/document_view.cc

```diff
@@ -209,7 +209,9 @@ mojo::Shell* DocumentView::GetShell() {
 
 void DocumentView::BeginFrame(base::TimeTicks frame_time) {
   if (sky_view_) {
-    sky_view_->BeginFrame(frame_time);
+    std::unique_ptr<LayerTree> layer_tree = sky_view_->BeginFrame(frame_time);
+    if (layer_tree->root_layer())
```

Comment:
seems weird that if BeginFrame() returns an empty tree you keep the old one. did you mean to check if |layer_tree| is null here (i.e. there was no View::m_scene, not that there was a View::m_scene with no layers in it?)